Timeseries_chapter.qmd

```diff
@@ -0,0 +1,200 @@
+---
+title: "Timeseries"
+bibliography: references.bib
+execute: 
+  freeze: auto
+output: 
+  html_document:
+   toc: true
+   toc_float: true
+---
+
+Temporal dependency means that values sampled at a given time can be related to the value sampled before. Therefore, the variable can be correlated to itself and create **pseudoreplication**. 
+
+
+Diverse packages on R are useful to manage time series, such as TSA (Time Series Analysis). 
+```{r, echo = F, error=FALSE}
+## Install and load the packages required
+
+library(tidyverse) # Data cleaning, ordering, etc.
+library(lubridate) # Helps manage the date format data
+library(ade4)      # Study ecological data
+library(nlme)      # glm and gls
+library(car)       # Type II anova.
+
+#To install older version of a package to use the arimax function.
+## why requiring old version ?
+# require(devtools)
+# install_version("TSA", version = "1.2.1", repos = "http://cran.us.r-project.org")
+# install_version("TSA", version = "1.2.1", repos = "https://pbil.univ-lyon1.fr/CRAN/")
+library(TSA)       # Time series analysis (autocorrelation tests)
+```
+
+### The dataset
+
+To understand better how to handle time series, data are used to illustrate the process from an experiment on vultures in time. 
+
+```{r}
+## Load the data
+## cette commande ne fonctionne qu'en local !!!
+## setwd("C:/Users/User/Onedrive/Bureau/MODE/M2/ASA/Le Gouar")
```

Comment:
@Raphaelle-Lebailly @amary35  @ascheubel  The data vautour.txt should be available on the repo, there is no way to have access to setwd("C:/Users/User/Onedrive/Bureau/MODE/M2/ASA/Le Gouar") from Github